Microsoft.Toolkit.Services/Services/MicrosoftGraph/MicrosoftGraphAuthenticationHelper.cs

```diff
@@ -4,12 +4,12 @@
 
 using System;
 using System.Linq;
-#if WINRT
+#if WINRT || WINDOWS_UWP
```

Comment:
This is provided by the compiler❔

Reply:
WINDOWS_UWP is a symbol in the default UWP project types and also in the Extras. Not sure where WINRT was being defined, but it wasn't automatically picked up.

Reply:
That's good to know. WINRT is defined in the Microsoft.Toolkit.Services project.